test/Garnet.test/RespHashTests.cs

```diff
@@ -160,6 +160,19 @@ public void CanDoHExists()
             Assert.AreEqual(false, result);
         }
 
+        [Test]
+        public void CanDoHStrLen()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+            db.HashSet("user.user1", new HashEntry[] { new HashEntry("Title", "Tsavorite") });
+            long r = db.HashStringLength("user.user1", "Title");
+            Assert.AreEqual(9, r, 0);
+            r = db.HashStringLength("user.user1", "NoExist");
+            Assert.AreEqual(0, r, 0);
+            r = db.HashStringLength("user.user2", "Title");
+            Assert.AreEqual(0, r, 0);
+        }
```

Comment:
Would be good to add a wrong # of tokens test here as well

Reply:
Sorry, what do you mean by "wrong # of tokens"? Is it the wrong number of arguments? This is the interface offer by StackExchange.Redis and have constraint two arguments already.

Reply:
Yeah same as the last test you wrote for LightClient, you can execute it with db.Execute(...) instead of using the predefined API methods... (e.g. db.Execute("HSTRLEN hashset key1 key2"))

Reply:
Yeah, I have tried it and it shows that StackExchange.Redis has already checked the params number and threw an ArgumentOutOfRangeException in https://github.com/StackExchange/StackExchange.Redis/blob/main/src/StackExchange.Redis/CommandBytes.cs#L111.

So I add another test with LightClient.